### ISSUE TITLE
Glist Unbanning is not detecting /tempban

### ISSUE THREAD

**Issue by Wild1145** [TotalFreedom/TotalFreedomMod#2111]
In a test of the current 1.12 Alpha branch when running a /tempban wild1145 10mins test 

And then running a /glist purge  and a /glist unban wild1145  the ban I previously had made is not detecting on the /glist unban, however on a /banlist purge it is. 

This appears to be a bit of an edge case as trying /tempban wild1145 10mins test and then /glist unban wild1145 worked without issue, however its something we might want to look into, either making /glist purge also wipe bans, or ensuring that /glist can still see bans even after it is purged.

**Comment by ghost** [comment]
Glist purge should wipe bans, not all data. As this is not the case I believe implementing that will solve the issue.

**Comment by Wild1145** [comment]
@Commodore64x That is incorrect. /glist purge should purge all player data, that is the intended design functionality of the command...
